src/Service.Host/client/src/containers/CreateTariff.js

```diff
@@ -1,17 +1,25 @@
 ﻿import { connect } from 'react-redux';
-import { addTariff } from '../actions/tariff';
-import EditTariff from '../components/EditTariff';
+import tariffActions from '../actions/tariff';
+import Tariff from '../components/Tariff';
+import getSingleErrorMessage from '../selectors/fetchErrorSelectors';
+import tariffSelectors from '../selectors/tariffSelectors';
 
-const mapStateToProps = () => ({
-    tariff: {},
-    editStatus: 'create'
+const mapStateToProps = (state) => ({
+    item: {},
+    editStatus: 'create',
+    loading: tariffSelectors.getLoading(state),
+    errorMessage: getSingleErrorMessage(state),
+    snackbarVisible: tariffSelectors.getSnackbarVisible(state)
 });
 
```

Comment:
You could initialise and dispatch actions to set edit status and set the tariff item in state to a blank object. I don't think this matters too much but I think its nice to clear out stale items from state.

Reply:
Not sure I follow, is there an example in one of the other containers?

Reply:
If you have a look at the CreateVatCode container. It dispatches a setEditStatus and a create action, these set the edit status and the vatCode item in the redux store. Without doing this vatCode in the store could still have stale information (if you've been editing another vatCode). It still works without doing this as the component is only using its local state. I think it seems wrong to have stale information in the redux store.